modular_skyrat/modules/3516/code/3516.dm

```diff
@@ -25,6 +25,7 @@
 	rack_sound = 'modular_skyrat/modules/3516/sound/slide.ogg'
 	fire_sound_volume = 100
 	bolt_wording = "fuckin' slide"
+	reload_time = 0 //FAST AS FUCK BOIS!
```

Comment:
does this thing really need more buffs

Reply:
It's mostly to keep it's robohand effect